services/uploads/src/utils.js

```diff
@@ -96,6 +96,8 @@ function extractBucketFromApiEvent(s3Event) {
  */
 function generateSystemMessage(systemMessage) {
   let finalMessage = `--- ${systemMessage} ---`;
+  // eslint-disable-next-line no-console
+  console.log(finalMessage);
```

Comment:
Still need this?

Reply:
Without this we don't see any logs on what it's doing